common/lib/xmodule/xmodule/util/adaptive_learning.py

```diff
@@ -169,8 +169,13 @@ def get_student(self, student_uid):
         """
         url = self.generate_student_url(student_uid)
         payload = {'key_type': 'uid'}
-        response = requests.get(url, headers=self.request_headers, data=payload)
-        students = json.loads(response.content)
+        try:
+            response = requests.get(url, headers=self.request_headers, data=payload)
+            students = json.loads(response.content)
+        except requests.ConnectionError:
+            students = []
+            log.warning("Could not retrieve information about student "
+                        "identified by student_uid {student_uid}.".format(student_uid=student_uid))
```

Comment:
@macornwell Thanks for making this line shorter!

However, the new format doesn't follow edX's Python style guide for [breaking long lines](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines). Please review the style guide so we can avoid needing multiple review cycles due to style issues in the future.

The same comment applies to the remaining log messages that this PR adds. (I'll skip adding separate comments for those.)